include/iocore/eventsystem/IOBuffer.h

```diff
@@ -1236,9 +1236,7 @@ class MIOBuffer
 
   const char *_location = nullptr;
 
-  MIOBuffer(void *b, int64_t bufsize, int64_t aWater_mark);
-  // cppcheck-suppress noExplicitConstructor; allow implicit conversion
-  MIOBuffer(int64_t default_size_index);
```

Comment:
What's the reason for making this explicit now?

Reply:
Makes sense. It's strange someone suppressed the linter check before instead of fixing it.

Reply:
I would guess that code was written before the `explicit` keyword was available in whatever compiler they were using to build ATS at the time.